message Route{
	required uint32 id = 1;
	required uint32 nstops = 2;
	required uint32 ntrips = 3;
	required uint32 stopsidx = 4;
	required uint32 tripsidx = 5;
	optional string name = 6;
}
message StopTime{
	required int32 arrival = 1;
	required int32 departure = 2;
}

message Stop{
	required uint32 id = 1;
	required uint32 nroutes = 2;
	required uint32 ntransfers = 3;
	required uint32 routeidx = 4;
	required uint32 transferidx = 5;
	optional string name = 6;
}

message Transfer{
	required uint32 from = 1;
	required uint32 to = 2;
	required float time = 3; 
}

message Timetable {
	repeated Route routes = 1;
	repeated StopTime stop_times = 2;
	repeated uint32 route_stops = 3;
	repeated Stop stops = 4;
	repeated Transfer transfers = 5;
	repeated uint32 stop_routes = 6;	
}
